### DISCUSSION TITLE
Is this possible to create a crontab schedule to run every second? (Using Database Scheduler)

### DISCUSSION THREAD

**Discussion by MuhammadBilal1** [celery/django-celery-beat#467] [Q&A]
Hi,
I am using the Django Admin Panel to create a new task and that task needs to run every second or every 10 seconds.
But I am unable to accomplish that.
I need to set this time using the database scheduler, not from the code.

Tried the following and none of them worked:
1- 1/1 * * * *
2- */0.1 * * * *
3- */1 * * * *

![image](https://user-images.githubusercontent.com/7131058/123284355-a0032900-d525-11eb-9a9d-360d38760069.png)

**Comment by alan2yang** [comment]
关注一下这个话题，怎么没人回复？why celery beat crontab doest not support seconds?
